examples/tumbleweed/Dockerfile

```diff
@@ -59,6 +59,14 @@ RUN ARCH=$(uname -m); \
       sed \
       patch \
       iproute2 \
+      selinux-policy-targeted \
```

Comment:
These packages could probably be moved to docs, unless we want to run tests in enforcing mode?